_includes/footer.njk

```diff
@@ -1,9 +1,11 @@
     <footer id="footer">
-      <a href="/" class="bigger-link">The Rehydrate podcast</a>
```

Comment:
This is needed so that, if you happen to enter via an episode page or any other non-index page, you can get to the main site.

Reply:
The h1 at the top of the main layout already has a link back to `/`, personally I would never think to look at the footer to go back to the homepage.

Reply:
Yeah, but that's a bug. The link says something like "Hosts", which isn't going to make people think it's a link to home.

Also, the footer link isn't for people that are thinking "I want to find the home page"; it's to suggest that to visitors who are done reading a pag to whom that idea never occurred.

Reply:
What do you think about if the h1 should always say "Rehydrate" and there should be an optional h2 for episode titles, "hosts", etc..

Reply:
That doesn't address getting suggestions about where to go after you're done with the main content, but it's definitely worth fixing the misleading h1 link. And one possible fix is to remove it completely.

If we decide to have an h1 and h2 at the top, we need watch out for there being so much stuff up there that the critical content gets pushed down too far.

Reply:
Updated both:

Homepage (no pageHeading)

![image](https://user-images.githubusercontent.com/26289/100520029-872dc300-3150-11eb-8db1-f719db246eca.png)

Transcript:

![image](https://user-images.githubusercontent.com/26289/100520036-94e34880-3150-11eb-99e1-77189e15ce99.png)

Footer:

![image](https://user-images.githubusercontent.com/26289/100520047-aa587280-3150-11eb-9963-4d80f9a92b6e.png)